src/color_table.py

```diff
@@ -0,0 +1,85 @@
+import json
+import logging
+from pathlib import Path
+from typing import Dict, List
+
+import colormath.color_conversions
+import colormath.color_diff
+import colormath.color_objects
+
+from src.logger import init_logging
+
+log = logging.getLogger()
+
+
+class Color:
+    def __init__(self, r: int, g: int, b: int):
+        self.__rgb_checker(r, g, b)
+        self.rgb = colormath.color_objects.AdobeRGBColor(rgb_r=r, rgb_g=g, rgb_b=b)
+        self.lab = colormath.color_conversions.convert_color(
+            color=self.rgb,
+            target_cs=colormath.color_objects.LabColor)
+
+    @staticmethod
+    def __rgb_checker(r, g, b):
```

Comment:
Rename to a verb (for example, "__check_rgb")

Reply:
done